CONTRIBUTING.md

```diff
@@ -1,24 +1,104 @@
 # Contributing
 
-Thank you for contributing to the docs! We have a few guidelines which will allow your awesome PR to pass our CI checks and successfully merge.
+Thank you for contributing to the docs! We have a few guidelines which will allow your PR to pass our CI checks and successfully merge.
 
-The docs version numbering adheres to [Semantic Versioning](http://semver.org/). Please ensure that each commit in your pull request conforms to the following format.
+## Creating Commits in line with Semantic Versioning
 
+The docs version numbering adheres to [Semantic Versioning](http://semver.org/). Please include *at least one commit* in your PR that marks the change-type for the system. 
+
+This change-type can be specified by adding it as a prefix, i.e., `minor: Add some new feature` while creating your commit. This is so the PR can be automatically versioned and a changelog generated for it by using versionbot. You have the option to choose one of the 3 `Change-type` when proposing a file change. 
+
+- `patch` - For tiny changes that include typo fixes, updating screenshots, adding clarification notes, etc.
+- `minor` - For moderate changes that include updating packages, rewriting documentation, adding new features.
+- `major`- For substantial changes that include updating large parts of the documentation or migrating to new frameworks. 
+
+### Creating Commits using the GitHub UI
+
+For folks using the Github UI to create new pull requests, check out the following guides in order to get started. 
+
+1. [Editing files and proposing changes](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository) on a GitHub Repository. When committing changes, here's how you can add a `Change-type` as a prefix to the commit. Adding `patch:` before your commit will mark the change-type as `patch` successfully. 
```

Comment:
Is it just before the commit message or it can work with the PR message as well? What happens when a PR has multiple commits?

Reply:
Thanks @pipex for your 👀 on this. 

1. Yes, just before "at least" one commit message would do. No need to sign commits as well. I have added a screenshot here + command-line way too. 
1.5. No, it won't work as a PR message, if you haven't yet created a PR on GitHub (like before hitting the "Create Pull Request" button) you can still go back to the previous page and add it. I have done it several times today. 

2. Again, until and unless you have it on at least on one commit. It's good. 

Anything from what I said above, should I add it as a clarification in the document.

Reply:
Added it with some bold.